modules/graphql-entity-reviser.js

```diff
@@ -0,0 +1,176 @@
+
+const utils = require("./graphman-utils");
+const revisers = [
+    policiesReviser11100(),
```

Comment:
is it OK to keep version in the method names, we might need to same revision in future versions also

Reply:
I expect, we shouldn't change the existing revisers. Instead, we should always append the new revisers to the list.

Reply:
This is something like DB upgrades.

Reply:
OK